db/external_sst_file_ingestion_job.cc

```diff
@@ -389,14 +389,25 @@ Status ExternalSstFileIngestionJob::IngestedFilesOverlapWithMemtables(
 Status ExternalSstFileIngestionJob::AssignLevelForIngestedFile(
     SuperVersion* sv, IngestedFileInfo* file_to_ingest, bool* overlap_with_db) {
   *overlap_with_db = false;
-
   Arena arena;
   ReadOptions ro;
   ro.total_order_seek = true;
 
   Status status;
   int target_level = 0;
   auto* vstorage = cfd_->current()->storage_info();
+
+  // Always go to last level, fail if it's not empty
+  if (ingestion_options_.ingest_behind) {
+    int lvl = cfd_->NumberLevels() - 1;
+    if (vstorage->NumLevelFiles(lvl) > 0) {
+      return Status::InvalidArgument(
+        "Can't ingest_behind file to non-empty bottommost level!");
+    }
+    file_to_ingest->picked_level = lvl;
```

Comment:
Even though we have checked that `immutable_db_options_.use_seqno_zero_out == false` that does not mean that the DB from the very beginning used this option, a good thing to have is to check the files in all upper levels that overlap with the files we are trying to ingest, and making sure that there lowest sequence number is not zero